src/myWallet/wallet.lightning.service.js

```
@@ -0,0 +1,407 @@
+    if (sats === null || sats === undefined) return 0;
+
+    // Handle BigInt or number or string safely
+    const satsNumber = typeof sats === 'bigint' ? Number(sats) : Number(sats);
+
+    return satsNumber / 100_000_000;
+}
+
+export const getLightningBalance = async (phrase) => {
+    try {
+        const sdk = await connectToSdk(phrase);
+        const obj1 = Object.fromEntries(sdkMap);
```

ESLint

[error] Delete `····`

(prettier/prettier)